### ISSUE TITLE
bias

### ISSUE THREAD

**Issue by nationalflag** [wenxinxu/ResNeXt-in-tensorflow#8]
Don't add bias to the conv layer?

**Comment by ondrejbiza** [comment]
Batch normalization adds bias.
